src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests_ISwitchOperation.cs

```diff
@@ -2076,9 +2076,9 @@ void M(bool result, dynamic input, dynamic other)
 }
 ";
             var expectedDiagnostics = new[] {
-                // file.cs(8,18): error CS0150: A constant value is expected
+                // (8,18): error CS9135: A constant value of type 'dynamic' is expected
```

Comment:
> // (8,18): error CS9135: A constant value of type 'dynamic' is expected [](http://example.com/codeflow?start=15&length=72)

This one is another interesting scenario. I think we should stick to the old message for dynamic, object and System.ValueType. Please add tests for these scenarios that are not currently covered.

Reply:
Done!